RecoCTPPS/TotemRPLocal/python/ctppsLocalTrackLiteProducer_cfi.py

```diff
@@ -1,13 +1,12 @@
 import FWCore.ParameterSet.Config as cms
 
-ctppsLocalTrackLiteProducer = cms.EDProducer("CTPPSLocalTrackLiteProducer",
-    tagSiStripTrack = cms.InputTag("totemRPLocalTrackFitter"),
-    tagDiamondTrack = cms.InputTag("ctppsDiamondLocalTracks"),
+from RecoCTPPS.TotemRPLocal.ctppsLocalTrackLiteDefaultProducer_cfi import ctppsLocalTrackLiteDefaultProducer
```

Comment:
This shouldn't be a cfi file, but a cff one.
Then you can import without the need of modifying the config name.

Of course,, if you do so the replacement  ctppsLocalTrackLiteProducer_cfi.py -> ctppsLocalTrackLiteProducer_cff.py should be ported in all configs which use it:

PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py 
RecoCTPPS/Configuration/python/recoCTPPS_cff.py
RecoCTPPS/TotemRPLocal/test/ctpps_reco_cfg.py

(see https://cmssdt.cern.ch/lxr/search?_filestring=&_string=psLocalTrackLiteProducer_cf )